src/stores/SetupEncryptionStore.ts

```diff
@@ -220,11 +219,6 @@ export class SetupEncryptionStore extends EventEmitter {
         this.emit("update");
     }
 
-    public reset(): void {
-        this.phase = Phase.ConfirmReset;
-        this.emit("update");
-    }
-
     public async resetConfirm(): Promise<void> {
```

Comment:
I think we can get rid of this and `returnAfterReset` ?

Reply:
Removed in 64ba6a5f85